packages/react-components/src/components/MessageThread.tsx

```diff
@@ -576,18 +576,18 @@ export interface RichTextEditBoxOptions extends RichTextEditorOptions {
    * Optional callback to handle an inline image that's inserted in the rich text editor.
    * When not provided, pasting images into rich text editor will be disabled.
    */
-  onInsertInlineImage?: (imageUrl: string, imageFileName: string, messageId: string) => void;
+  onInsertInlineImage?: (imageAttributes: Record<string, string>, messageId: string, imageFileName?: string) => void;
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
   /**
-   * Optional callback to remove the image upload or delete the image from server before sending.
+   * Optional callback invoked after inline image is removed from the UI.
    */
-  onCancelInlineImageUpload?: (imageId: string, messageId: string) => void;
+  onRemoveInlineImage?: (imageAttributes: Record<string, string>, messageId: string) => void;
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
   /**
    * Optional Record of type {@link AttachmentMetadataInProgress}
-   * to render inline images being inserted in the MessageThread's edit boxes.
+   * to provide progress and error info for inline images inserted in the MessageThread's edit boxes.
```

Comment:
I think we don't show any progress, do we? If so, this doc might be confusing

Reply:
> I think we don't show any progress, do we? If so, this doc might be confusing

We don't show visual progress for now. In GA, there will be some visual indication.
We use the progress to show the in progress errorBar through.
This prop is providing the progress info that we're using.